services/movie_session.py

```diff
@@ -41,4 +41,13 @@ def update_movie_session(
 
 
 def delete_movie_session_by_id(session_id: int) -> None:
-    MovieSession.objects.get(id=session_id).delete()
+    MovieSession.objects.get(pk=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    return list(
+        MovieSession.objects
+        .filter(pk=movie_session_id)
+        .prefetch_related("tickets")
+        .values(row=F("tickets__row"), seat=F("tickets__seat"))
```

Comment:
or you could do something like this 
`list(get_movie_session_by_id(movie_session_id).tickets.values("row", "seat"))`